App/Author/author.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+
+class Author
+  attr_accessor :id, :firstname, :lastname, :items
+
+  def initialize(firstname, lastname)
+    @firstname = firstname
+    @lastname = lastname
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
+
+  def convert_to_hash
+    {
+      id: id,
+      firstname: firstname,
+      lastname: lastname
+    }
+  end
+
+  def save_authors_to_file(authors)
+    data = authors.map(&:convert_to_hash)
+    File.write('authors.json', JSON.pretty_generate(data))
+  end
+
+  def self.load_authors_data
+    return [] unless File.exist?('authors.json') && !File.empty?('authors.json')
+
+    author_data = JSON.parse(File.read('authors.json'))
+    author_data.map do |data|
+      author = Author.new(data['firstname'], data['lastname'])
+      author.id = data['id'] # Set the author's ID
+      author
+    end
+  end
+
+  def self.list_all_authors
+    authors_data = load_authors_data
+    authors_data.each do |author|
+      puts "ID: #{author.id}, Name: #{author.firstname}, LastName: #{author.lastname} "
+    end
+  end

```

Comment:
- [ ] @Team when I delete the author.js file and add a new author and try to access the list of authors it doesn't show the authors as shown below 
![image](https://github.com/supreeti/Ruby-Capstone-CatalogOfMyThings/assets/61408860/3c9d1cdf-fb66-452a-86a1-d5d6fd2e4232)

Reply:
- [ ] @Team the same case as above happens when trying to view the games added 
![image](https://github.com/supreeti/Ruby-Capstone-CatalogOfMyThings/assets/61408860/1eb74d77-06a8-41aa-b50e-61d0ddf5d742)